wandb/cli/cli.py

```diff
@@ -1341,6 +1341,13 @@ def launch(
                 run_id=run_id,
                 repository=repository,
             )
+            if not run_async:
```

Comment:
what happens if it is run_async, do we just not care about the run?

Reply:
yeah, in the sdk context (run = launch(...)) the user can check the SubmittedRun object status directly. I guess we could simplify this to just if `run.get_status().state == "failed"`.  I'll do that, it will work in some async cases (but introduce race conditions) and all sync cases.